manager/manager.go

```diff
@@ -30,7 +30,7 @@ type TagFilter struct {
 // Filter returns the ec2.Filter representation of the TagFilter.
 func (t *TagFilter) Filter() *ec2.Filter {
 	return &ec2.Filter{
-		Name:   aws.String(fmt.Sprintf("tag:%s", t.Key)),
+		Name:   aws.String(fmt.Sprintf("%s", t.Key)),
```

Comment:
This is a breaking change you might want to view.  Tag filter is a subset of filter and I have opened it up to any filter type.  Refactoring the name should be done to drop the reference to 'Tag'

Reply:
Makes sense! Couple of notes on this:
- We can just drop `fmt.Sprintf` here since we are not formatting the key anymore.
- We need to move the old `fmt.Sprintf` to [here](https://github.com/itsdalmo/ssm-sh/blob/master/command/list-instances.go#L29) then to avoid breaking list instances.

After this PR is merged with the above changes, we could rename the current `--filter` flag to `--filter-tag` and then support both filtering arbitrary things, and filtering on tags. So thats good 👍

Reply:
While I see that a specfic switch could be useful for stacks... it seems a bit redundant considering that Tag:Name works and is already documented in the AWS CLI.